content/_includes/embed.macros.njk

```diff
@@ -323,7 +323,13 @@ params:
 ) -%}
 {%- if src -%}
 {%- set src = src if ('://' in src) else ('/assets/images/' + src) -%}
-<img src="{{ src }}" alt="{{ alt }}" {{ utility.show_attrs(attrs) }}/>
+<img
+  srcset="{{src | imgSuffix('320')}} 320w, {{src | imgSuffix('550')}} 550w"
+  sizes= "(max-width: 550px) calc(100vw - 120px), 550px"
+  src="{{ src }}"
```

Comment:
In the MDN examples, they duplicate the default image in both places. I'm going to look into that a bit, and see if it's necessary… 

I also wonder if we want to define our largest size as well (assuming many images will be larger than `550` by default), maybe something like `1200`?

Reply:
ok, Eric Portis pointed me to [the spec](https://html.spec.whatwg.org/multipage/images.html#introduction-3) which says that `src` is ignored when using `w`. So I think we do need to represent larger images in the `srcset` somehow.

Reply:
I think using `src` is helpful for unsupported browsers. Creating default image size at 1200 make sense.  I'm (doubtful, but) curious if this affects images that are naturally smaller. Would it load images that are too big for what we need?

Reply:
- Yeah, we don't want to remove `src`, but we do need to duplicate that path in the `srcset`
- If possible, I think we should only shrink images, and never enlarge them.